playground/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+    "compilerOptions": {
+        "target": "ES2020",
+        "module": "ES2020" /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */,
+        "lib": ["DOM", "ES2020"],
+        "moduleResolution": "node",
+        "strict": true,
+        "noEmitOnError": true,
+        "baseUrl": "./",
+        "paths": {
+            "@/*":["../modular-particle-system/src/*"]
```

Comment:
see other comment https://github.com/Risto-Paasivirta/ParticleSystem/pull/77/files#r820143968, you don't need that, you will use the correct import